apps/src/templates/curriculumCatalog/CurriculumCatalogCard.jsx

```diff
@@ -49,7 +49,7 @@ const CurriculumCatalogCard = ({
     quickViewButtonDescription={i18n.quickViewDescription({
```

Comment:
@dmcavoy Is the screenreader description of the button still okay? It reads "View details about {course_name}"

Reply:
I think that still makes sense as they will view the details of the course on another page. (Not sure how screenreader friendly that page they will open is tho)

Reply:
Agreed that is hard. Sam is working on some key student flows over the next several months and we can keep making sure the things we touch meet accessibility guidelines. Over time we will get to a better place. Please continue to be the voice for this stuff. I've learned a lot and its so important.